.run/NussknackerApp.run.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="NussknackerApp" type="Application" factoryName="Application">
-    <option name="ALTERNATIVE_JRE_PATH" value="corretto-11" />
+    <option name="ALTERNATIVE_JRE_PATH" value="corretto-17" />
```

Comment:
Are you sure about this? I suposse that has been change in rsr PR according Java 17?

Reply:
Currently, our main supported java version is java 17.

Reply:
It just wasn't changed in the run configurations in the repo, my IDE changed it when I executed NussknackerApp